src/assets/custom/css/_sass/_website-navigation-menu.scss

```diff
@@ -81,18 +82,35 @@
       border-width: 0 2px 2px 0;
       border-color: $grey--dark;
       margin: 3px;
-      transform: rotate(45deg);
-      transition: border-color 0.5s;
+      transition: border-color 0.5s, transform 0.5s, top 0.5s;
+      position: relative;
     }
   }
+  @include large-and-extra-large {
+    border: 0 solid transparent;
+    transition: border-bottom-width 0.5s;
```

Comment:
Would it be clearer to use `margin-bottom` ?

Reply:
Yes, yes it would https://github.com/codurance/site/pull/1921/commits/e70241191aacf1f75b2393624bc30466882981bd